src/ray/core_worker/core_worker.cc

```diff
@@ -1963,6 +1963,10 @@ std::optional<std::vector<rpc::ObjectReference>> CoreWorker::SubmitActorTask(
   }
 
   auto actor_handle = actor_manager_->GetActorHandle(actor_id);
+  if (actor_handle->IsActorStateUnsubscribed()) {
+    // Lazy subscribe.
+    actor_manager_->SubscribeActorState(actor_id);
```

Comment:
The check for whether we are subscribed seems kind of meaningless since SubscribeActorState is idempotent anyway.

Reply:
> The check for whether we are subscribed seems kind of meaningless since SubscribeActorState is idempotent anyway.

Yes, indeed. I will remove this check.